src/ts/component/menu/dataview/option/list.tsx

```diff
@@ -234,6 +238,12 @@ const MenuOptionList = observer(class MenuOptionList extends React.Component<I.M
 		const { onChange, maxCount } = data;
 
 		let value = Relation.getArrayValue(data.value);
+
+		if (value.includes(id)) {
+			this.onValueRemove(id);
```

Comment:
this is not correct, onChange handler is not called.

Reply:
it's called from onValueRemove method. I split this logic to increase readability, can combine them back of course.

Reply:
looks ugly and not readable